tests/integration/test_rex.py

```diff
@@ -77,49 +76,29 @@ def test_literal_null(c):
     SELECT NULL AS "N", 1 + NULL AS "I"
     """
     )
-    df = df.compute()
 
     expected_df = pd.DataFrame({"N": [pd.NA], "I": [pd.NA]})
     expected_df["I"] = expected_df["I"].astype("Int32")
-    assert_frame_equal(df, expected_df)
+    assert_eq(df, expected_df)
 
 
-def test_random(c, df):
-    result_df = c.sql(
-        """
-    SELECT RAND(0) AS "0", RAND_INTEGER(1, 10) AS "1"
-    """
-    )
-    result_df = result_df.compute()
+def test_random(c):
+    query = 'SELECT RAND(0) AS "0", RAND_INTEGER(0, 10) AS "1"'
 
-    # As the seed is fixed, this should always give the same results
-    expected_df = pd.DataFrame({"0": [0.26183678695392976], "1": [8]})
-    expected_df["1"] = expected_df["1"].astype("Int32")
-    assert_frame_equal(result_df, expected_df)
+    result_df = c.sql(query)
+    expected_df = c.sql(query)
```

Comment:
I'm guessing the idea here is to verify that multiple executions with the same seed give the same result. Could we rename `expected_df` to something else since I was a little confused why `expected_df` was re-running the same query initially.